packages/admin/src/pages/build-application/[applicationId]/[sectionId]/question-type/index.page.tsx

```diff
@@ -5,44 +5,61 @@ import {
-import CustomLink from '../../../../components/custom-link/CustomLink';
-import Meta from '../../../../components/layout/Meta';
+import CustomLink from '../../../../../components/custom-link/CustomLink';
+import Meta from '../../../../../components/layout/Meta';
 import ResponseType, {
   ResponseTypeLabels,
-} from '../../../../enums/ResponseType';
-import { getApplicationFormSummary } from '../../../../services/ApplicationService';
-import { postQuestion } from '../../../../services/QuestionService';
+} from '../../../../../enums/ResponseType';
+import { getApplicationFormSummary } from '../../../../../services/ApplicationService';
+import { postQuestion } from '../../../../../services/QuestionService';
 import {
   addFieldsToSession,
   getSummaryFromSession,
   getValueFromSession,
-} from '../../../../services/SessionService';
-import { QuestionSummary } from '../../../../types/QuestionSummary';
-import callServiceMethod from '../../../../utils/callServiceMethod';
-import { getSessionIdFromCookies } from '../../../../utils/session';
-import QuestionTypeHint from './components/QuestionTypeHint';
-import {
-  getErrorPageParams,
-  questionErrorPageRedirect,
-} from './newQuestionServiceError';
+} from '../../../../../services/SessionService';
+import { QuestionSummary } from '../../../../../types/QuestionSummary';
+import { getSessionIdFromCookies } from '../../../../../utils/session';
+import QuestionTypeHint from '../components/QuestionTypeHint';
+import { questionErrorPageRedirect } from '../newQuestionServiceError';
+import ResponseTypeEnum from '../../../../../enums/ResponseType';
+import QuestionPageGetServerSideProps from '../../../../../utils/QuestionPageGetServerSideProps';
 
 type RequestBody = {
   responseType: ResponseType;
   _csrf?: string;
 };
 
-const WORD_LIMIT_MAP = {
-  [ResponseType.ShortAnswer]: 300,
-  [ResponseType.LongAnswer]: 5000,
-};
+const redirectQuestionType = [
+  ResponseType.Dropdown,
+  ResponseType.MultipleSelection,
+  ResponseType.LongAnswer,
+];
+
+function getRedirect(
+  responseType: ResponseType,
+  applicationId: string,
+  sectionId: string
+) {
+  const REDIRECT_MAP = {
+    [ResponseType.Dropdown]: `/build-application/${applicationId}/${sectionId}/question-options`,
+    [ResponseType.MultipleSelection]: `/build-application/${applicationId}/${sectionId}/question-options`,
+    [ResponseType.LongAnswer]: `/build-application/${applicationId}/${sectionId}/question-type/add-word-count`,
+  };
+  return REDIRECT_MAP[responseType as keyof typeof REDIRECT_MAP];
+}
 
-export const getServerSideProps: GetServerSideProps = async ({
-  params,
-  req,
-  res,
-}) => {
+const SHORT_QUESTION_WORD_LIMIT = 300;
+
+function getRadio(responseType: ResponseType) {
+  if (responseType === ResponseType.Dropdown) return 'Multiple choice';
+  if (responseType === ResponseType.MultipleSelection) return 'Multiple select';
+  return '';
+}
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
```

Comment:
Might be worth shifting getServerSideProps to a separate file, and importing then exporting it here